src/sentry/api/endpoints/organization_group_index.py

```diff
@@ -104,6 +105,12 @@ def get(self, request, organization):
         if not projects:
             return Response([])
 
+        if len(projects) > 1 and not features.has(
+                'organizations:sentry10', organization, actor=request.user):
```

Comment:
The frontend checks for the `global-views` feature as that is the billing related feature that toggles whether or not a customer's plan allows them to view data from more than one project at a time.

Reply:
updated, ty!